release/ray_release/test_automation/state_machine.py

```diff
@@ -79,8 +79,7 @@ def _init_ray_buildkite(cls):
     def get_release_blockers(cls) -> List[github.Issue.Issue]:
         repo = cls.get_ray_repo()
         blocker_label = repo.get_label(WEEKLY_RELEASE_BLOCKER_TAG)
-
-        return repo.get_issues(state="open", labels=[blocker_label])
+        return list(repo.get_issues(state="open", labels=[blocker_label]))
```

Comment:
is this conversion safe (e.g. could it be that only the first page becomes the list)

Reply:
based on pygithub docs:
https://pygithub.readthedocs.io/en/latest/utilities.html#github.PaginatedList.PaginatedList

PaginatedList is a iterable, so it can be directly converted to a list, and will contain all the items.

one risk is that, if the list is too long, it can run out of memory or time out, but not very likely for our use case here. I can add a limit guard though if you would like me to add one.